db/config.hh

```diff
@@ -61,6 +62,29 @@ struct seed_provider_type {
     }
 };
 
+/*
+ * This type is a glue between configuration in ms
+ * and duration
+ */
+struct duration_in_ms_type {
```

Comment:
Why not use duration directly?

Reply:
The issue is that the configuration is in ms, so the config needs to create some kind of glue between that (number of ms) and duration.

I think that translating int directly to duration is confusing. when you will look at the code, you'll see `duration` and you need to guess what are the units.

Using this kind of type makes it clearer.

Reply:
std::chrono::milliseconds names the units directly (and duration has an std::ratio parameter that defines them)